modules/share/config.py

```diff
@@ -0,0 +1,8 @@
+def can_build(env, platform):
+    return env["platform"] in ["ios"]
```

Comment:
Fully platform dependent module with bound API can't be properly documented, and inconvenient to use from the code. We might want to add some way around it, but since this functionality can be implemented at least on 3 platforms (macOS, iOS, Android), it probably should have always available platform independent base class subclasses for specific platforms.

Reply:
for now it available on ios and android, not sure about macos, for android I have plugin, which trying to convert into part of godot core

Reply:
macOS API - https://developer.apple.com/documentation/appkit/nssharingservicepicker?language=objc

Reply:
This can be added as a next step, maybe even windows right now have this kind of api